lib/index.ts

```diff
@@ -1,30 +1,32 @@
 /* global FormData */
+import type { core, JSONSchema } from '@balena/jellyfish-types';
```

Comment:
What is being used from that `core` namespace? If it's just contracts that's fine but if it's something else you should replace those with the actual types.

Reply:
For `jellyfish-types` exceptionally it's fine to have deep imports. Makes it easier to see what is actually being imported. We'll be removing this module anyway.

Reply:
It's just `core.Contract`.